apps/server/src/modules/task/uc/task-copy.uc.spec.ts

```diff
@@ -119,12 +126,13 @@ describe('task copy uc', () => {
 			};
 		};
 
-		it('should throw if copy feature is deactivated', async () => {
-			Configuration.set('FEATURE_COPY_SERVICE_ENABLED', false);
-			const { course, user, task, userId } = setup();
-			await expect(uc.copyTask(user.id, task.id, { courseId: course.id, userId })).rejects.toThrowError(
-				InternalServerErrorException
-			);
+		describe('feature is deactivated', () => {
+			it('should throw InternalServerErrorException', async () => {
+				Configuration.set('FEATURE_COPY_SERVICE_ENABLED', false);
+				await expect(uc.copyTask('user.id', 'task.id', { courseId: 'course.id', userId: 'test' })).rejects.toThrowError(
```

Comment:
We always want to have a blank line between Arrange, Act and Assert. It's missing everywhere in this file. Not your change but maybe you could fix it.